frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableDetail/CollapsibleHeader/CollapsibleHeader.module.css

```diff
@@ -7,6 +7,7 @@
   border-bottom: 1px solid var(--global-border);
   background-color: var(--pane-background);
   cursor: pointer;
+  position: sticky;
```

Comment:
Good, but there is a duplication of the following:

https://github.com/liam-hq/liam/blob/646235c8a4e0d86b13bf780e897a0b44bcf908ee/frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableDetail/CollapsibleHeader/CollapsibleHeader.module.css#L21-L30

Media query may not necessary, please merge it?